### PR TITLE
Added GMV Syncromatics DX Templates

### PR DESCRIPTION
**Pull request by stevenmwhite**
Added documentation based on the GMV Syncromatics templates.

### CODE REVIEW

**Review by thzinc — APPROVED** [review]

**Comment by stevenmwhite** [comment]
What's the process here after a PR is approved? Does the owner/maintainer then "merge" them into the master?

**Comment by thzinc** [comment]
@stevenmwhite, yes, that is the process. I hadn’t gotten around to it until now. Thanks for your patience!

**Pull request merged**
